### PR TITLE
169 values mixed with challenges

### PR DESCRIPTION
**Pull request by CaptainSifff**
This works on the  values.
I think the most notable omission in our values section id GSP.
Thereby we now kind of have the structure, that we have two Research values, two SE values, and two Team values,
mirroring that all three aspects are equal-weight pillars. I've tried to use the values in two mire places.
The part for the challenges is still not quite resolved.

### CODE REVIEW

File: competencies.md [hunk 1]
```diff
@@ -320,6 +326,8 @@ Therefore, we categorise the competencies into software engineering skills,
 research skills, and team skills with a particular focus on the software and
 research cycle and the scientific process. The generic skills are relevant in a
 broad setting and form the foundation for specific specialisations.
+These competencies have been chosen, to make RSEs contribute to an open and inclusive research
```
Comment (MakisH) [inline competencies.md:329]:
```suggestion
These competencies have been chosen in order to make RSEs contribute to an open and inclusive research
```

File: competencies.md [hunk 2]
```diff
@@ -276,39 +281,40 @@ Independent from the encoding into the respective national law in an RSE's juris
...
 This imbalance is even more pronounced in software engineering with a majority of developers identifying as white male [@StackOverflow2022].
-In their work RSEs might frequently find themselves in a position
-to encourage, mentor and empower people who gravitate towards software-related occupations.
-In this capacity, they should be aware of the diversity problem and help to mediate it
-whenever they have the chance to do so.
+Thereby to promote their values of an honest, open  and inclusive research space they should be aware of
+the diversity problem and help to mediate it whenever they have the chance to do so.
 
-## The scientific community
+### The scientific community
```
Comment (MakisH) [inline competencies.md:301]: I wonder if this is the best heading. Is the community a challenge?

Maybe:

```suggestion
### Long-term effects to the scientific community
```

**Review by MakisH — APPROVED** [review]
There is now a clear bridge between values and challenges: The subsection split did the trick.

Cross-checking the connection between values and challenges:

3.1.1 (personal data) does **not** directly connect to one of the six values, but I think it originates from the guide for good scientific practice (how to handle data of people participating in experiments).

3.1.2 (mentoring) directly connects to the "work towards a fair and inclusive environment" and "promote these values".

3.1.3 (scientific community) connects at least to the good scientific practice guidelines, to the "commit to objectivity", to the "promote openness", and to "develop software that adheres to current best practices".

3.1.4 (emerging, LLMs) connects to "objectivity and fact-based, honest research conclusions" and to "judge independently and maintain professional integrity".

No values remain unreferenced from the challenges.

I think the connection to the generic skills is implicitly clear anyway.

File: competencies.md [hunk 3]
```diff
@@ -276,39 +281,40 @@ Independent from the encoding into the respective national law in an RSE's juris
 the right to information privacy is internationally recognised as a fundamental human right,
 e.g. in the European Convention on Human Rights [@CouncilOfEuropeProtocol1988; @Hirvela2022].
 RSEs need to be aware of this topic's importance
-and deal with tensions that might arise with researchers' desire for trouble-free sharing of data.
+and deal with tensions that might arise with researchers' desire for trouble-free sharing of data, thereby expecting openness about the research process,
+versus the integrity expectations of the society towards IT systems.
 Handling personal data also has ramifications for information security considerations during the software development process.
 Data protection is a difficult subject and RSEs should notice when they need to consult external expertise for example when dealing with
 special topics such as cryptography or re-identification attacks (e.g. [@Sweeney2002]).
```
[the remark this answers is not in the document]
Comment (CaptainSifff) [reply competencies.md]: OK, I'm now using a reidentifacation literature review paper.

File: competencies.md (hunk 2 above)
Comment (CaptainSifff) [reply competencies.md]: I went with "Shaping digital science".

**Pull request merged**